modules/exploits/linux/http/fusionpbx_php_editor_cmd_exec.rb

```diff
@@ -0,0 +1,135 @@
+####################################################################
+# This module requires Metasploit: https://metasploit.com/download #
+#  Current source: https://github.com/rapid7/metasploit-framework  #
+####################################################################
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+        "Name" => "FusionPBX =< 4.5.10 Authenticated Remote Code Execution",
+        "Description" =>  %q{
+          Authenticated Remote Code Execution in FusionPBX web appliances.
+          Affected versions: =< 4.5.10
+          By using the on site PHP-Editor function users can overwrite PHP files on the web server
+          with a trojan horse and get a shell as www-data by navigating to the changed PHP file location.
+          Vendor verified: 11/27/2019
+          Vendor patched: 12/09/2019
+          Public disclosure: 01/20/2020
+        },
+        "License" => MSF_LICENSE,
+        'Author' => [
+          'enjloezz', # Metasploit Module
+          'TheCyberGeek' # Discovery and Metasploit Module
+        ],
+        'References' =>
+        [
+            ['URL','https://github.com/fusionpbx/fusionpbx/releases'],
+            ['CVE','2019-19369']
```

Comment:
I saw a tweet saying this was on EDB, but I can't seem to find it.  Was it ever put there, if so add it here as well.

Reply:
No not for this. I only have CVE-2019-16405 on EDB.